en/docs/install-and-setup/upgrading-wso2-api-manager/260-to-410/upgrading-from-260-to-410.md

```diff
@@ -0,0 +1,761 @@
+# Upgrading API Manager from 2.6.0 to 4.1.0
+
+The following information describes how to upgrade your API Manager server **from APIM 2.6.0 to 4.1.0**.
+
+!!! note
+    Before you follow this section, see [Upgrading Process]({{base_path}}/install-and-setup/upgrading-wso2-api-manager/upgrading-process) for more information.
+
+!!! Attention
+    If you are using WSO2 Identity Server (WSO2 IS) as a Key Manager, first you have to follow the instructions in [Upgrading WSO2 IS as the Key Manager to 5.11.0]({{base_path}}/install-and-setup/upgrading-wso2-is-as-key-manager/upgrading-from-is-km-570-to-is-5110) instead of below steps.
+
+!!! Attention
+    As the on-premise analytics data cannot be migrated to the Cloud, you need to maintain the old analytics server and keep the UI running for as long as you need that data (e.g., 3 months) after migrating to the new version of analytics in WSO2 API-M 4.1.0.
+    
+!!! note "If you are using PostgreSQL"
```

Comment:
Better to move this step to DB configuration section

Reply:
Note about Postgres was already there in DB section, therefore removed it from here and moved note on oracle with commit https://github.com/wso2/docs-apim/pull/5985/commits/9725faeeeda65a8cc7570354f8fbc9ce24110e1e